cfme/tests/services/test_config_provider_servicecatalogs.py

```diff
@@ -124,3 +124,74 @@ def test_retire_ansible_service(appliance, catalog_item, request, job_type):
     assert order_request.is_succeeded(method='ui'), msg
     myservice = MyService(appliance, catalog_item.name)
     myservice.retire()
+
+
+@pytest.mark.manual
+@test_requirements.tower
+@pytest.mark.tier(1)
+def test_config_manager_override_extra_vars_dialog_vsphere():
+    """
+    1. add tower 2.4.3 provider and perform refresh
+    2. Go to job templates
+    3. Create service dialog from third_job_template
+    - this template is bound to vsphere55 inventory
+    - simple_play_4_dockers.yaml playbook is part of this template
+    - this playbook will touch /var/tmp/touch_from_play.txt
+    - into /var/tmp/touch_from_play_dumped_vars.txt all variables
+    available during play run will be dumped
+    - this includes also variables passed from Tower or CFME
+    - this project is linked with Tower and Vsphere55 credentials
+    - Vsphere55 credentials are used when inventory is retrieved
+    - Tower credentials are the creds used to login into VM which will be
+    deployed
+    - Vsphere template used for VM deployment must have ssh key "baked" in
+    -
+    Prompt for Extra variables must be enabled.
+    4. Add Vsphere55 provider into CFME and perform refresh
+    5. Create new Catalog
+    6. Add new catalog item for Vsphere vcentre - VMWare
+    7. give it name, display in catalog, catalog,
+    8 Provisioning entry point:
+    /Service/Provisioning/StateMachines/ServiceProvision_Template/CatalogI
+    temInitialization
+    Request info tab:
+    Name of template: template_Fedora-Cloud-Base-23-vm-tools_v4 (this
```

Comment:
These are sub points of Request Info tab right ? 
Can we start them with '-' as you did above in line 137 to 145